frontend/scripts/react-components/profile-root/profile-root.scss

```diff
@@ -1,5 +1,59 @@
 @import 'styles/settings';
 
+.c-profile-root-new {
+  overflow: hidden;
+
+  .profile-root-search-container {
+    padding: 90px 0;
+
+    @media only screen and (max-width: $breakpoint-foundation-small) {
+      padding: 60px 0;
+    }
+  }
+
+  .profile-root-actions {
+    margin-top: 70px;
+
+    @media only screen and (max-width: $breakpoint-foundation-small) {
+      margin-top: 50px;
+    }
+
+    text-align: center;
+  }
+
+  .browse-button {
+    width: 270px;
```

Comment:
No custom sizes, let's use the design system. cc. @fannycc and Pablo H.

Reply:
This is because the regular button doesn't fit all the text

Reply:
Then it might be a bug, the styles should be adding padding not width.